pkg/innerring/processors/netmap/process_peers.go

```diff
@@ -107,14 +107,29 @@ func (np *Processor) processUpdatePeer(ev netmapEvent.UpdatePeer) {
 
 	var err error
 
+	if ev.Maintenance() {
+		err = np.nodeStateSettings.MaintenanceModeAllowed()
+		if err != nil {
+			np.log.Info("prevent switching node to maintenance state",
+				zap.Error(err),
+			)
```

Comment:
Shouldn't we move node to OFFLINE in this case?

Reply:
"Offline" leads to exclusion from the network map, while "maintenance" not. I don't think such an implicit switch would be expected.